baseline-error-prone/build.gradle

```diff
@@ -36,14 +36,30 @@ dependencies {
     compileOnly 'com.google.auto.service:auto-service'
 }
 
-// Incorrectly identifies tests as junit4 usage
-tasks.checkJUnitDependencies.enabled = false
-tasks.withType(JavaCompile) {
-    options.errorprone {
+tasks.withType(JavaCompile).configureEach {
+    sourceCompatibility = 11
+    options.compilerArgs += [
+        '--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.model=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.processing=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED'
```

Comment:
should we just set this in `gradle.properties`?

Reply:
It does not, I spent couple hours trying to make it work and nothing made sense. Since with toolchains with fork the jvm for compilation add this to gradle.properties doesn't fix compilation